hadoop-ozone/dist/src/main/smoketest/basic/links.robot

```diff
@@ -114,9 +114,12 @@ Key list passthrough
                         Should Contain              ${source_list}    key2
 
 Key delete passthrough
-                        Execute                     ozone sh key delete --skipTrash ${target}/link1/key2
-    ${source_list} =    Execute                     ozone sh key list ${source}/bucket1 | jq -r '.[].name'
-                        Should Not Contain          ${source_list}    key2
+                        Execute                     ozone sh key delete ${target}/link1/key2
+    ${source_list} =    Execute                     ozone sh key list ${source}/bucket1 | jq -r '.[] | select(.name | startswith(".Trash")) | .name'
+                        Should Contain Any          ${source_list}     .Trash/hadoop    .Trash/testuser    .Trash/root
+                        Should contain              ${source_list}    key2
+    ${result} =         Execute                     ozone sh key list ${source}/bucket1 | jq -r '.[] | select(.name | startswith(".Trash") | not) | .name'
+                        Should Not contain          ${result}         key2
```

Comment:
Please save output of `ozone sh key list ${source}/bucket1` to a file, and apply multiple checks on that file (`jq ... bucket.list`) to avoid multiple round-trips to Ozone.

Reply:
Done